src/Apps/Artist/Routes/Combined/ArtistCombinedRoute.tsx

```diff
@@ -101,6 +101,30 @@ const ArtistCombinedRoute: React.FC<
     scrollToSection()
   }, [section, jumpTo, waitUntil, location.pathname, router])
 
+  // When the page is accessed via a URL that contains a hash to a jump link
+  // anchor -- as in the case of the 301 redirects we've set up --
+  // we similarly scroll to the appropriate section and clean up the URL
```

Comment:
This code is similar but not exactly the same as the other `scrollToSection` so it seemed reasonable to duplicate it here.